api/observability/v1/output_types.go

```diff
@@ -210,15 +210,48 @@ type BaseOutputTuningSpec struct {
 
 	// MinRetryDuration is the minimum time to wait between attempts to retry after delivery a failure.
 	//
+	// A duration string is a possibly signed sequence of decimal numbers, each with optional fraction
+	// and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or
+	// "µs"), "ms", "s", "m", "h".
+	//
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Minimum Retry Duration"
 	MinRetryDuration *time.Duration `json:"minRetryDuration,omitempty"`
 
 	// MaxRetryDuration is the maximum time to wait between retry attempts after a delivery failure.
 	//
+	// A duration string is a possibly signed sequence of decimal numbers, each with optional fraction
+	// and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or
+	// "µs"), "ms", "s", "m", "h".
+	//
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Maximum Retry Duration"
 	MaxRetryDuration *time.Duration `json:"maxRetryDuration,omitempty"`
+
+	// MaxBufferSize is the maximum size in bytes allowed for buffering unsent data to an output.
+	// It is written as a [Kubernetes quantity], for example: 128974848, 129e6, 129M,  123Mi
+	//
+	// NOTE: There are trade-offs to changing buffer sizes.
+	// The defaults are calculated to provide good performance in most cases.
+	// It is not recommended to set this field unless you have done tests
+	// to determine a value that performs better than the defaults in your specific setting.
+	// See also []
+	//
+	// The following are some trade-offs to bear in mind:
+	//
+	// - MaxBufferSize is per-output, add up all the outputs for the total impact per-node.
```

Comment:
it can be event more severe; it is sum per output of ALL deployed CLF instances.  We should consider how if the existing alert we have is sufficient and if there is anything in the operator we can do to guard against master nodes having issues

Reply:
If we are to add buffer size then we also need to warn users that this is a cumulative issue where ALL forwarders deployed that declares disk buffer is an issue.  We run the risk of consuming too much node disk and destabilizing every workload on an individual node.  This is exacerbated on master nodes